pkg/service/healthcheck/service.go

```diff
@@ -71,8 +65,7 @@ type Service struct {
 
 	cacheMu sync.Mutex
 	// fields below are protected by cacheMu
```

Comment:
Maybe change **fields** to singular since now there is only one thing protected by the mutex.